### ISSUE TITLE
List of Technicians in a form

### ISSUE THREAD

**Issue by McklRgt** [pluginsGLPI/formcreator#2113]
Hello, 
I use Glpi 9.5.2 and the plugIn formcreator 2.11.0
I want ot create a form that contain a list of technicians (with defaut selection of tehcnician connected to Glpi) but I don't find a choice to obtain a list of users who are declared as Technicians. 
Is it possible, if yes how can I do that ?
Thanks eveybody !

**Comment by btry** [comment]
Hi

You must create a target ticket, if not done already. Next, edit the target ticket. You will find a lot of settings. Near the bottom of the modal form, you can configure actors.

Le 1 février 2021 15:58:50 GMT+01:00, McklRgt <notifications@github.com> a écrit :
>Hello, 
>I use Glpi 9.5.2 and the plugIn formcreator 2.11.0
>I want ot create a form that contain a list of technicians (with defaut
>selection of tehcnician connected to Glpi) but I don't find a choice to
>obtain a list of users who are declared as Technicians. 
>Is it possible, if yes how can I do that ?
>Thanks eveybody !
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/pluginsGLPI/formcreator/issues/2113

-- 
Envoyé de mon appareil Android avec Courriel K-9 Mail. Veuillez excuser ma brièveté.

**Comment by McklRgt** [comment]
Thanks for your answer but my problem is before that, in the "Questions" Tab where I'm not able to generate a list with only the technicians in it. 
Do you speak french, because I'm french ?

**Comment by btry** [comment]
Yes I speak French, but it is better to speak English, to allow more people to read the thread.

I understand that you want to add technicians from a question of the form. The plugin does not allows for now to limit the items in its dropdowns with the criteria you need. There is need to detect where is used a question and such scenarios are under consideration (this would be the best solution to automatically determine criterias).

This is very likely possible, and this is a feature. You may want to contact us to get a quote.

Le 1 février 2021 16:57:17 GMT+01:00, McklRgt <notifications@github.com> a écrit :
>Thanks for your answer but my problem is before that, in the
>"Questions" Tab where I'm not able to generate a list with only the
>technicians in it. 
>Do you speak french, because I'm french ?
>
>-- 
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/pluginsGLPI/formcreator/issues/2113#issuecomment-770959954

-- 
Envoyé de mon appareil Android avec Courriel K-9 Mail. Veuillez excuser ma brièveté.
